.github/workflows/checking.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           args: --workspace
 
       - name: Run cargo check (with serde)
-        if: ${{ matrix.toolchain != '1.42.0' }} # The following args don't work on older versions of Cargo
+        if: ${{ matrix.toolchain != '1.49.0' }} # The following args don't work on older versions of Cargo
```

Comment:
If we're bumping MSRV we probably don't need this condition, which was only put in because the old versions of Cargo couldn't handle the `cargo check` command with serde.